docs/validation.md

```diff
@@ -616,21 +616,220 @@ While this behaviour is unexpected (issues [#129](https://github.com/ajv-validat
 }
 ```
 
-The schema above is also more efficient - it will compile into a faster function.
+The schema above is also more efficient - it will compile into a faster function. Alternatively, you may use the [option `removeUnevaluated`](./validation.md#removing-unevaluated-properties).
 
 ### Removing unevaluated properties
 
-With [option `removeUnevaluated`](./api.md#options) you can filter data during the validation.
+With [option `removeUnevaluated`](./api.md#options) you can filter data during the validation. It is only compatible with draft 2019-09.
+
+This option modifies original data. It removes every property or item that would raise an error based on the keywords `unevaluatedProperties` and `unevaluatedItems`. It will first remove unevaluated properties then unevaluated items.
+
+Compared to `removeAdditional`, it will always take into account all sub-schemas that were valid for this data instance : 
+
+
+- all subschemas schemas in `allOf` and `$ref` keywords
+- valid sub-schemas in `oneOf` and `anyOf` keywords
+- sub-schema in `if` keyword
+- sub-schemas in `then` or `else` keywords that were applied based on the validation result by `if` keyword.
+
+
+
+
+Example 1 : remove unevaluated properties 
+
+```javascript
+
+const ajv = new Ajv({removeUnevaluated: true})
```

Comment:
```suggestion
const AJV = require('"ajv/dist/2019") //unevaluated is only available in AJV 2019
const ajv = new Ajv({removeUnevaluated: true, unevaluated: true})
```

I also think you need to add `unevaluated: true` to even turn on this validation, though it might be the default (I will check this later)

Reply:
It works by default. I added the ajv import to make sure people understand it only works with ajv/dist/2019.

Reply:
They use the boolean unevaluated:true inside the ajv2019 constructor. 

![image](https://user-images.githubusercontent.com/45051524/104797576-3b30ef00-578d-11eb-9d7c-ff900fb33ddf.png)